source/Octostache.Tests/FiltersFixture.cs

```diff
@@ -503,5 +502,165 @@ public void TrimWithInvalidOptionDoesNoting()
             var result = Evaluate(@"#{foo | Trim Both}", new Dictionary<string, string> { { "foo", "Octopus Deploy" } });
             result.Should().Be("#{foo | Trim Both}");
         }
+
+        [Fact]
+        public void UriPartNoArgument()
```

Comment:
I like how comprehensive these tests are. However, I think the readability would be improved by using `InlineData` something like this:

```
[InlineData("https://octopus.com/docs", "UriPart", "UriPart: no argument given")]
[InlineData("https://octopus.com/docs", "UriPart bar", "UriPart: argument 'bar' not supported")]
[InlineData("https://octopus.com/docs", "UriPart AbsolutePath", "/docs")]
public void UriPart(string inputUrl, string inputFilterExpression, string expectedOutput)
{
   Evaluate(@"#{foo | UriPart}", new Dictionary<string, string> { { $"foo | {inputFilterExpression}", inputUrl } }).Should().Be(expectedOutput);
}
```

Reply:
Fixed in the latest commit.